corehq/motech/repeaters/models.py

```diff
@@ -852,6 +852,8 @@ def wrap(cls, data):
             cls_ = cls.get_class_from_doc_type(data['doc_type'])
             if cls_:
                 return cls_.wrap(data)
+            elif data['doc_type'] == 'Repeater' and settings.UNIT_TESTING:
```

Comment:
Would it be possible to patch this wrap method in tests rather than doing a `settings.UNIT_TESTING` check here? Trying to avoid having test code mixed with production code.

Reply:
This has been removed and in tests I have used SQLCaseRepeater rather than SQLRepeater (https://github.com/dimagi/commcare-hq/pull/30348/commits/f69d3add04004b6eb8d7295fec963ea10ea3adbf)for which we would have to patch a lot of things to make it work.
The reason I went with it was in production too we would not be having any code that creates a Repeater of `doc_type` Repeater.